force_wfmanager/wfmanager_results_task.py

```diff
@@ -0,0 +1,215 @@
+import logging
+
+from pyface.api import ImageResource
+from pyface.tasks.action.api import SMenuBar, SMenu, TaskAction, SToolBar
+from pyface.tasks.api import Task, TaskLayout, PaneItem
+
+from traits.api import Bool, Instance, List, on_trait_change
+
+from force_bdss.api import (IFactoryRegistryPlugin, Workflow)
+
+from force_wfmanager.central_pane.analysis_model import AnalysisModel
+from force_wfmanager.central_pane.graph_pane import GraphPane
+from force_wfmanager.left_side_pane.results_pane import ResultsPane
+from force_wfmanager.TaskToggleGroupAccelerator import (
+    TaskToggleGroupAccelerator
+)
+
+log = logging.getLogger(__name__)
+
+
+class WfManagerResultsTask(Task):
+    """Task responsible for running the Workflow and displaying the results."""
+
+    id = 'force_wfmanager.wfmanager_results_task'
+    name = 'Results'
+
+    #: Workflow model.
+    workflow_m = Instance(Workflow, allow_none=False)
+
+    #: Analysis model. Contains the results that are displayed in the plot
+    #: and table
+    analysis_m = Instance(AnalysisModel, allow_none=False)
```

Comment:
Same comment.

Reply:
Yeah good point! I think we'd just got too used to calling them that!